src/gesv_mixed.cc

```diff
@@ -110,10 +114,13 @@ void gesv_mixed(
     const Layout layout = Layout::ColMajor;
 
     bool converged = false;
-    const int itermax = 30;
     using real_hi = blas::real_type<scalar_hi>;
     const real_hi eps = std::numeric_limits<real_hi>::epsilon();
     const scalar_hi one_hi = 1.0;
+
+    int64_t itermax = get_option<int64_t>( opts, Option::MaxIterations, 30 );
+    double tol = get_option<double>( opts, Option::Tolerance, eps*std::sqrt(A.m()) );
```

Comment:
I think we could already replace the `std::sqrt` as suggested here [https://github.com/icl-utk-edu/blaspp/issues/43](blas++ issue)

Reply:
I mean, `A.m()` will always be an `int64_t`.  So, unless we're planning to add support for templating indices on non-standard integer types, we won't run into the issue from icl-utk-edu/blaspp#43 .